zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -890,7 +898,8 @@ angular.module('zeppelinWebApp')
         var d = pivotDataToD3ChartFormat(p, true).d3g;
 
         $scope.chart[type].x(function(d) { return d.label;})
-          .y(function(d) { return d.value;});
+          .y(function(d) { return d.value;})
+          .donut(donutFlag);
```

Comment:
Try extending donut out of pie and use it here

nv.models[‘donut’] = function(){
	return nv.models[‘pieChart’]().donut(true);
}